iroh-gateway/src/client.rs

```diff
@@ -40,21 +48,23 @@ impl Client {
             .metrics
             .hist_ttfb
             .observe(start_time.elapsed().as_millis() as f64);
-        let res = resolver.resolve(p).await.map_err(|e| e.to_string())?;
-        let res = res.pretty(rpc_client).await.map_err(|e| e.to_string())?;
+        let res = self.resolver.resolve(p).await.map_err(|e| e.to_string())?;
+        let reader = res.pretty(rpc_client.clone());
+        let stream = ReaderStream::new(reader);
+        let body = StreamBody::new(stream);
 
         info!("resolved: {}", path);
         state
             .metrics
             .tts_file
             .set(start_time.elapsed().as_millis() as u64);
-        let n = res.len() as u64;
-        state
-            .metrics
-            .bytes_per_sec_out
-            .set(n / start_time.elapsed().as_secs().max(1));
-        state.metrics.bytes_streamed.inc_by(n);
-        Ok(res.into())
+        // let n = res.len() as u64;
```

Comment:
you can drop this completely, we don't need this counter. Alternatively I'll do it on the next metrics cleanup

Reply:
removed